src/gridcoin/voting/registry.cpp

```diff
@@ -449,9 +804,10 @@ void PollRegistry::AddPoll(const ContractContext& ctx)
     const auto payload = ctx->SharePayloadAs<PollPayload>();
     std::string poll_title = payload->m_poll.m_title;
 
-    if (ctx->m_version == 1) {
-        boost::to_lower(poll_title);
-    }
+    // The title used as the key for the m_poll map keyed by title, and also checked for duplicates, should
+    // not be case-sensitive, regardless of whether v1 or v2+. We should not be allowing the insertion of two v2 polls
+    // with the same title except for a difference in case.
+    boost::to_lower(poll_title);
```

Comment:
Alternatively:
```
// The title used as the key for the m_poll map keyed by title, and also checked for duplicates, should
// not be case-sensitive, regardless of whether v1 or v2+. We should not be allowing the insertion of two v2 polls
// with the same title except for a difference in case.
std::string poll_title = boost::to_lower_copy(payload->m_poll.m_title);
```

Reply:
This is cycy's code, so I am trying not to screw with it too much, since he has a branch which is going to make a bunch of changes coming up, and I don't want to create more merge conflicts. You are right that either is more concise, but I think the original change is ok. I simply removed the v1 conditional from the to_lower so that all polls are keyed in lower case. I am going to run a test on testnet from a build prior to this PR and with this PR trying to insert a poll with the same title but differing case and see what happens.

Reply:
Oki, then he can fix it in his upcoming PR.